doc/integrator/authentication.rst

```diff
@@ -10,6 +10,22 @@ obtained from the "auth ticket" cookie set in the request.
 The policy is created, and added to the application's configuration, in the
 application's main ``__init__.py`` file.
 
+In the ``.env.sample`` we can congigure it with the following variable:
```

Comment:
In the file ``.env.sample``, you can configure the policy with the following variables: